app/views/people/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<table>
+
+  <% @displaypeople.each do |person| %>
+    <tr>
+      <% if person.tasks.where(comp_status: "unread") == 1 %>
```

Comment:
I think you could DRY this up and use this conditional for the sole purpose of adding an "s" to "suggestion" and setting that to a variable. This would help you avoid having duplicate `<td>` entries.